keps/sig-storage/20190204-kubernetes-csi-release-process.md

```diff
@@ -166,9 +171,21 @@ For each component under kubernetes-csi (`external-attacher`,
 `csi-driver-host-path`, `csi-lib-utils`, etc.), these Prow jobs need
 to be defined:
 - `kubernetes-csi-<component>-pr: presubmit job
-- `kubernetes-csi-<component>-build: a postsubmits job that matches against
+- `kubernetes-csi-<component>-build`: a postsubmits job that matches against
   `v*` branches *and* tags (see https://github.com/kubernetes/test-infra/pull/10802#discussion_r248900281)
 
+In addition, for the `csi-driver-host-path` repo some more periodic
+jobs need to be defined:
+- `kubernetes-csi-stable`: deploys and tests the current hostpath
+  example (`csi-driver-host-path/deploy/stable`) from the master
+  branch on the latest Kubernetes development version
+- `kubernetes-csi-canary-1.13`: deploys and tests the canary hostpath
```

Comment:
maybe replace "1.13" with "<k8s-release>", and then define how many releases back we want to keep testing.

Also, how many branches of the hostpath driver do we want to test as well?

Reply:
Also, the canary jobs should be careful and use a specific image hash and not the canary tag, otherwise, the image may change in the middle of the run.

Reply:
I've included that in the description of the job.